### ISSUE TITLE
Chrome - "in-page-icon dropdown on username-fields" broken since 1.7.5

### ISSUE THREAD

**Issue by wadabum** [keepassxreboot/keepassxc-browser#1205]
## Expected Behavior
after clicking show a list of accounts right next to the input-field

## Current Behavior
![animation of the error](https://i.imgur.com/sWRtdnU.gif)
'nothing after click' + JS-Error:
```
keepassxc-browser.js:1505 Uncaught (in promise) TypeError: Cannot read property 'some' of null
    at keepassxc-browser.js:1505
    at Array.filter (<anonymous>)
    at Object.kpxc.updateTOTPList (keepassxc-browser.js:1505)
    at async CredentialAutocomplete.showList (autocomplete.js:82)
```

## Possible Solution
oddity: after restarting the browser the dropdown shows up ONCE,  
but fails "in the same frame twice", so does it fail in new tab/different urls

## Steps to Reproduce (for bugs)

1. have "in-page icons enabled"
2. have keypass connected
3. browse to site with saved pw
4. click icon for dowpdown
5. reload page
6. try clicking the icon again -> no dropdown.

## Debug info

KeePassXC - 2.5.4
KeePassXC-Browser - 1.7.5
Operating system: Win
Browser: Chrome 88.0.4324

**Comment by wadabum** [comment]
To note:
- I continue to see my accounts & autofill works when clicking "the extension-icon in chrome"  
so data-wise things are fine

**Comment by wadabum** [comment]
just to confirm, it updated and `fixed in 1.7.6`

much appreciated! 👍
